Plutarch/Api/V1/Value.hs

```diff
@@ -334,28 +456,30 @@ pnormalize = phoistAcyclic $
       pif (intData #== zeroData) (pcon PNothing) (pcon $ PJust intData)
 
 -- | Assert the value is properly sorted and normalized.
-passertSorted :: Term s (PValue _ _ :--> PValue 'Sorted 'NonZero)
+passertSorted :: forall anyKey anyAmount s. Term s (PValue anyKey anyAmount :--> PValue 'Sorted 'NonZero)
```

Comment:
Why `forall`?

Reply:
I forgot to remove this lol - was playing around with the type variable ordering